playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -0,0 +1,154 @@
+
+6. Run commands:
+
+
+```console
+# Create a new authentication configuration for GCP Project with the created user account
+gcloud init
+
+# Command imports new user account credentials into Application Default Credentials
+gcloud auth application-default login
+
+# Navigate to 01.setup folder
+cd playground/terraform/infrastructure/cloudbuild-manual-setup/01.setup/
+
+# Run terraform scripts
+terraform init -backend-config="../../../environment/{environment_name}/state.tfbackend"
+terraform plan -var-file="../../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 2. Connect GitHub repository and Cloud Build
+
+**Note:** Ensure correct `region` is set in [Cloud Build Triggers](https://console.cloud.google.com/cloud-build/triggers) page before proceeding further.
+
+Follow [Connect to a GitHub repository](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github)
+to connect the GitHub repository `https://github.com/{github_repository_owner}/{github_repository_name}` and `{github_branch_name}` with Cloud Build.
+
+## 3. Setup the Google Cloud Build triggers
+
+The `playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders` provisions:
+- Cloud Build triggers to build and deploy Beam Playground.
+
+#### To execute the module
+
+
+```
+# Navigate to playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders folder
+cd ../02.builders
+
+# Run terraform scripts
+terraform init -backend-config="../../../environment/{environment_name}/state.tfbackend"
+terraform plan -var-file="../../../environment/{environment_name}/terraform.tfvars"
+terraform apply -var-file="../../../environment/{environment_name}/terraform.tfvars"
+```
+
+## 4. Running the first Cloud Build trigger to deploy Playground infrastructure
+
+1. Navigate to [GCP Console Cloud Build Triggers](https://console.cloud.google.com/cloud-build/triggers) page. Choose the region (Example: us-central1).
+2. Open Trigger: `Playground-infrastructure-trigger`.
+3. Scroll down to `Advanced` - `Substitutions variables` and Click on `+ ADD VARIABLE`.
+4. Assign values for the next variables:
+    - `_ENVIRONMENT_NAME` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; # *Created "environment_name"*
+    - `_DNS_NAME`  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; *# Your DNS record name*
+    - `_LOGS_BUCKET_NAME` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; *# Your GCP logs bucket name `logs_bucket`*
+5. Click `Save` and Run the trigger `Playground-infrastructure-trigger`.
+
+6. Once Playground infrastructure has been deployed, please navigate to
+   [Playground deployment README](https://github.com/apache/beam/tree/master/playground/terraform#deploy-playground-infrastructure) and execute step #2:
+   `Add following DNS A records for the discovered static IP address`
+
+## 5. Running the second Cloud Build trigger to deploy Playground to GKE
```

Comment:
@ruslan-ikhsan consider rephrasing, e.g.:
Run Cloud Build `Playground-to-gke-trigger` to deploy Playground to GKE